sjb/config/common/test_cases/origin_release.yml

```diff
@@ -8,4 +8,6 @@ extensions:
       script: |-
         export OS_BUILD_IMAGE_ARGS=''
         hack/build-base-images.sh
-        make release
\ No newline at end of file
+        OS_BUILD_ENV_PRESERVE="_output/local" hack/env make build-rpms
+        hack/build-images.sh
```

Comment:
The current `make release` cannot run inside `hack/env` because it uses `hack/env` already I think. `make release` with `build-rpms` works fine inside `hack/env`.